api/_mixins/ParentNode__Element.json

```diff
@@ -116,10 +116,17 @@
             "firefox_android": {
               "version_added": "4"
             },
-            "ie": {
-              "version_added": "9",
-              "notes": "Internet Explorer 6, 7 and 8 supported it, but erroneously includes <code>Comment</code> nodes."
-            },
+            "ie": [
+              {
+                "version_added": "9"
+              },
+              {
+                "version_added": "6",
+                "version_removed": "9",
+                "partial_implementation": true,
+                "notes": "Erroneously includes <code>Comment</code> nodes."
```

Comment:
I tested http://software.hixie.ch/utilities/js/live-dom-viewer/?saved=8928 in IE 7 and 8 to confirm this. If one is being pedantic, this note is incorrect, because comments were then [`HTMLCommentElement`](https://docs.microsoft.com/en-us/openspecs/ie_standards/ms-html401e/0fd60633-29a6-4377-97f1-d0466e2e21ce) objects, which then explains why they're included.

Reply:
hm, then calling this partial support is probably stretching it a bit too far?